yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -127,7 +130,7 @@ export function makeEmptyAccumulatedData(seed = 1): CombinedAccumulatedData {
     fr(seed + 13),
     range(KERNEL_NEW_COMMITMENTS_LENGTH, seed + 0x100).map(fr),
     range(KERNEL_NEW_NULLIFIERS_LENGTH, seed + 0x200).map(fr),
-    range(KERNEL_PRIVATE_CALL_STACK_LENGTH, seed + 0x300).map(fr),
+    new Array(KERNEL_PRIVATE_CALL_STACK_LENGTH).fill(Fr.ZERO), // private call stack must be empty
```

Comment:
Looking at the function name, shouldn't all of these fields actually be empty?

Reply:
It's a badly named function (by me). Some items need to be empty and others not. It's coming back to that discussion around crafting a range of inputs in TS to satisfy circuit 'modes'.